### PR TITLE
Feature/79 add temporary support link to footer of front page

### PR DESCRIPTION
**Pull request by gazaston**
screen shot 2018-04-17 at 17 31 58

### CODE REVIEW

**Comment by gazaston** [comment]
[Add temporary support link to footer of front page](https://trello.com/c/rnL5nNWy/79-add-temporary-support-link-to-footer-of-front-page)

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -39,4 +39,10 @@
         %span= msg
     = yield
 
+- content_for :footer_top do
+  %p
+    Problems using this service? Email
+    =mail_to 'isobel.croot@digital.education.gov.uk'
```
Comment (despo) [inline app/views/layouts/application.html.haml]: Can we extract this to an environment variable and reuse so it's not stored in the codebase.

File: app/views/shared/_beta_banner.html.haml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.flash.notice
+  .small This service is still in development, so for now only lists jobs in select schools in Cambridgeshire and the North East. When it launches, the full service will list jobs in schools throughout England.
+  .small
+    Interested in listing jobs at your school? Email
+    =mail_to 'isobel.croot@digital.education.gov.uk'
```
Comment (despo) [inline app/views/shared/_beta_banner.html.haml]: same as above

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (tahb) [reply app/views/layouts/application.html.haml]: Agreed.

**Pull request merged**